src/main/java/com/fruitshop/dao/DataBaseManager.java

```diff
@@ -0,0 +1,13 @@
+package com.fruitshop.dao;
+
+import com.fruitshop.model.Fruit;
+import java.util.Map;
+
+public interface DataBaseManager {
+
+    void addToDB(String typeOfFruit, Fruit fruit);
+
+    Fruit getFromDB(String keyFruit);
+
+    Map<String,Fruit> getAllDB();
```

Comment:
You can name you crud method jsut `add`, `get` and `getAll`. We know that we are working with db here seeing the name